cms/signals/slug_update_fix.py

```diff
@@ -0,0 +1,32 @@
+"""
+This is a temporary fix that should be replaced with an adequate bug fix to the Page logic.
```

Comment:
I'd opt to fix this properly rather than bring in a temporary fix that never actually gets fixed.